src/main/java/com/communityratesgames/model/UserModel.java

```diff
@@ -73,6 +73,10 @@ public UserModel toModel(User user) {
         return um;
     }
 
+    public String toJMS(User user) {
+        return user.getUserName();
+    }
```

Comment:
Can you motivate this? Why not just use `user.getUserName()` directly instead of creating a function for it, and why put it in `UserModel` and not making it static despite not using any of the variables within `UserModel`? In fact, this is better off in `User`, as that is where it belongs.

Reply:
Man skickar ett rent TextMesage, inte modell eller entitet. Enklast så för nu iallafall

Reply:
Dessutom, static vill jag så lite som möjligt använda när jag inte behöver det

Reply:
Förklarar fortfarande inte varför funktionen är i `UserModel` och inte i `User`. Du använder inte ens någon variabel i `UserModel`, utan istället tar in en `User`, kallar en getter på den och returnerar den utan att ens röra `UserModel` över huvud taget.
Den tillhör inte `UserModel`. Den tillhör `User` eftersom vi arbetar objektorienterat, och det är där den ska ligga.

Reply:
flytta den då

Reply:
Du kan skriva så här i `User` istället:
```
public String toJMS() {
    return this.getUserName();
}
```